### PR TITLE
DDF-4409 Fixing CSS to keep text breaking out of popovers

### PR DESCRIPTION
**Pull request by aperullo**
Link to 2.13.x PR: #4152

#### What does this PR do?
Adds a CSS fix to keep long texts from breaking out their textboxes in popovers.

#### Who is reviewing it? 
@pvargas 
@mdang8 
@emmberk 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@clockard 

#### How should this be tested?
Verify in downstream project.
Implement a popover with a long unbreaking string of text. Verify it does not break out of the popover.

#### What are the relevant tickets?
[DDF-4409](https://codice.atlassian.net/browse/DDF-4409)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/registry-admin-remote-ui/src/main/webapp/less/styles.less [hunk 1]
```diff
@@ -45,6 +45,10 @@ table.align-middle {
     color: red;
     display: block;
   }
+  .popover-content {
+    overflow-wrap: break-word;
```
Comment (andrewkfiedler) [inline ui/packages/registry-admin-remote-ui/src/main/webapp/less/styles.less]: Looks like putting both overflow-wrap and word-wrap is redundant. 

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Comment (aperullo) [reply ui/packages/registry-admin-remote-ui/src/main/webapp/less/styles.less]: Alright, removed that property and retested, thank you!

**Comment by aperullo** [comment]
build now

**Comment by andrewkfiedler** [comment]
No need to hero since it's a direct port of https://github.com/codice/ddf/pull/4152

**Pull request merged**